README.md

```diff
@@ -200,3 +152,61 @@ glRotatef(0.0, 0.0, 1.0, 0.0)
 glutSolidTeapot(0.5)
 ```
 
+---
+
+### Building & Testing
```

Comment:
We have Building & Testing - and then Testing again?` I suggest maybe renaming this to `Building jextract` (for symmetry) and then `Testing jextract`.